pkg/agent/kube.go

```diff
@@ -123,13 +123,23 @@ func KubeEvents(kubeEnv *KubeEnv, gimletHost string, agentKey string) {
 						pod.Namespace == deployment.Namespace {
 						for _, event := range events.Items {
 							if event.Type == "Warning" && (event.InvolvedObject.Name == pod.Name || event.InvolvedObject.Name == deployment.Name) {
+								first := event.FirstTimestamp.Time
+								firstInDuration := time.Since(first).Seconds()
+								frequency := float64(event.Count) / firstInDuration
```

Comment:
This calculates a per second event rate.
Technically correct, but we won't be able to alert on more advanced rules, like 10 times in an hour, and alike.
We need the raw data on the server so we can make practical rules.